scripts/deploy_preview.sh

```diff
@@ -45,11 +45,15 @@ kubectl config set-context --current --namespace=default
 echo "Getting the pods"
 kubectl get pods
 
-echo "Delete previously created namespace"
-kubectl delete ns $NAMESPACE || echo "true"
+if [ ! -z "$RECREATE" ]
```

Comment:
Prefer something like this.

1. `-n` is the inverse of `-z`.
2. Double square brackets are almost _always_ better than single square brackets.
3. Using `${RECREATE-}` instead of just `$RECREATE`, means that is `nounset` is set in the shell, and this env variable is not defined, the script will break. Just a good habit/practice. 🙂

```suggestion
if [[ -n "${RECREATE-}" ]]
```